File: 250316/물의 상태/state-of-water.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int temperture = sc.nextInt();

        if (temperture >= 100) {
            System.out.println("ice");
        }
        else if (temperture >= 0) {
            System.out.println("water");
        }
        else {
            System.out.println("ice");
        }
    }
}

File: 250316/물의 상태/state-of-water.cpp
#include <iostream>
using namespace std;

int main() {

    int temperture = 0;
    cin >> temperture;

    if(temperture >= 100) {
        cout << "vapor";
    }
    else if (temperture >= 0) {
        cout << "water";
    }
    else {
        cout << "ice";
    }
    return 0;
}